src/Documents/Session/IAttachmentsSessionOperations.ts

```diff
@@ -62,8 +62,15 @@ export interface IAttachmentsSessionOperations {
      */
     get(entity: object, name: string, callback: AbstractCallback<AttachmentResult>): Promise<AttachmentResult>;
 
-    //TBD AttachmentResult GetRevision(string documentId, string name, string changeVector);
-    
+    /**
+     * Returns the revision attachment by the document id and attachment name.
+     * @param documentId Document Id
+     * @param name Name of attachment
+     * @param changeVector Change vector
+     * @return Attachment
+     */
+    getRevision(documentId: string, name: string, changeVector: string): Promise<AttachmentResult>;

```

Comment:
callback api plz?

Reply:
Please notice another method in this class doesn't have callback overrides.

Reply:
done

Reply:
done